core/examples/http_bench.js

```diff
@@ -121,8 +163,8 @@ async function serve(rid) {
 }
 
 async function main() {
-  Deno.core.setAsyncHandler(handleAsyncMsgFromRust);
-
+  Deno.core.setAsyncHandler(Op.handleAsyncMsgFromRust);
+  initOps(Deno.core.getOps());
```

Comment:
I don't like all the indirection for simple logic like this. Remove the initOps function and put its body here.